src/vulkan/mod.rs

```
@@ -368,6 +375,20 @@ impl MemoryBlock {
             } else {
                 alloc_info
             };
+            #[cfg(windows)]
+            let mut export_info = vk::ExportMemoryAllocateInfoKHR::default()
+                .handle_types(vk::ExternalMemoryHandleTypeFlags::OPAQUE_WIN32_KHR);
+            #[cfg(all(unix, not(target_vendor = "apple")))]
+            let mut export_info = vk::ExportMemoryAllocateInfoKHR::default()
+                .handle_types(vk::ExternalMemoryHandleTypeFlags::OPAQUE_FD_KHR);
+
+            // On other platforms you can't create an external capable allocator, so this would be unreachable
+            #[cfg(any(windows, all(unix, not(target_vendor = "apple"))))]
```

I think on other platforms then we should fail with an error rather then silently failing to create an exportable texture.